helm/templates/jmx-configmap.yaml

```diff
@@ -16,6 +16,12 @@ data:
         name: "cp_kafka_schema_registry_jetty_metrics_$1"
       - pattern : 'kafka.schema.registry<type=master-slave-role>([^:]+):'
         name: "cp_kafka_schema_registry_master_slave_role"
+      - pattern : 'kafka.schema.registry<type=jersey-metrics>([<>])*(.+)\.(([^-]*)-(.+)):'
+        name: "cp_kafka_schema_registry_endpoint"
+        labels:
+          name: "$2"
+          category: "$4"
+          metric: "$5"
       - pattern : 'kafka.schema.registry<type=jersey-metrics>([^:]+):'
```

Comment:
you can remove this one?

Reply:
Removed prefixes.
But still keeping the existing regex to capture any non endpoint level metrics, only `endpoint.metric` are taken care by new one. Have kept older below in order for lower precedence.